src/shapes/itext.class.js

```diff
@@ -264,10 +264,10 @@
      * Prepare and clean the contextTop
      */
     clearContextTop: function(skipRestore) {
-      if (!this.isEditing || !this.canvas || !this.canvas.contextCursor) {
```

Comment:
Looks like contextTextbox only exists if you are using a highlight_canvas. For a 'normal' canvas this will will be undefined and will exit here. I'd suggest setting a contextTextbox = this.contextTop in the normal canvas class as well.

Reply:
Done.